indicator of page# for users.index

Snail51 opened the pull request:
addresses #20

when viewing the user.index view, if the `page=` URI parameter is set, a visual indicator is placed on the screen saying what page the viewer is on

Snail51 committed:
indicator of page# for users.index

addresses #20

when viewing the user.index view, if the `page=` URI parameter is set, a visual indicator is placed on the screen saying what page the viewer is on

drraccoony commented:
Looks right, but will review more soon

drraccoony commented on resources/views/users/index.blade.php:
```diff
@@ -28,6 +28,10 @@ class="block rounded-md bg-white px-3 py-2 text-center text-sm font-semibold tex
                             <p class="mt-2 text-sm text-gray-700">Listing of all users/volunteers within the organzation including their email, sector/dept, status, and hours.</p>
+                            @if(null !==request('page'))
+                                <p class="mt-2 text-sm text-orange-700"><x-heroicon-s-magnifying-glass class="w-4 inline"/> Currently showing <span class="underline">Page #{{$users->currentPage()}}</span> of {{$users->lastPage()}}.
+                                <a class="text-blue-600" href="{{route('users.index')}}">Clear Search</a>
```
You shouldn't be showing Clear search on this, its context is not relevant.

Snail51 commented:
On slack you state

> :no_entry_sign: MR 23 rejected ("Clear Search" context)

By this, do you mean you want the "Clear Search" `<a href="#">` element removed from that line? Such that only actual searches say "Clear Search"?

drraccoony commented:
Why is "Clear Search" contextually important to the page number? If you have a search term, **and** are on page 3, you're going to see that twice.

Snail51 committed:
remove "Clear Search" link from page info

the message that shows up saying what page you are on no longer has a "Clear Search" option, as this is not a search.

Snail51 commented:
`Clear Search` is not contextually important to the page number. This link has been removed in d1ed71996451090f28037e658d59fc321de4a904.
